src/EngineManager.php

```diff
@@ -30,6 +31,14 @@ public function createAlgoliaDriver()
         ));
     }
 
+    public function createElasticsearchDriver()
+    {
+        return new Engines\ElasticsearchEngine(Elasticsearch::create()
```

Comment:
ok, noticed that option now, could be a great option for custom setups. thanks @nWidart